python/test/nex2ecl.py

```diff
@@ -0,0 +1,92 @@
+import datetime
+import unittest
+import os
+import tempfile
+import shutil
+import nex
+import ecl.ecl as ecl
+
+
+class TestLoad(unittest.TestCase):
+    def setUp(self):
```

Comment:
I think you should reorganize the tests slightly:

1. The `plt` and `plt_delatyed` are for-all-prictical purposes static and could be implemented using the `setupClass()`method instead:

```python
@classmethod
def setupClass(cls):
     cls.plt = nex.load()
     cls.plt_delayed = 
```

2. I *really* think the change of cwd - and then the subsequent changing back should be isolated in one function. I *highly* recommend the use of a context variable. Personally I am very familiar with the `TestAreaContext` from `libecl`:

```python
from ecl.test import TestAreaContext
...
...
def test_something(self):
     with TestAreaContext("name_of_test"):
            # Do something - write to file
     --> Here the temporary area is deleted.
``` 
there are many approaches to this - @pgdr has implemented some nice decorators for use in everest testing.

3. When looking at your tests it is only `test_file_io` which actually needs a scratch area.

Reply:
This is fixed in RP https://github.com/Statoil/nex/pull/13